build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.1.0'
     automatticTracksVersion = '3.0.0'
-    gutenbergMobileVersion = 'v1.99.0'
+    gutenbergMobileVersion = '5889-debb12289b51074bae1834cec59688e8fdd4eb91'
```

Comment:
I believe this should point to Gutenberg Mobile's trunk, so please make sure to merge Gutenberg's PR first, then update this before merging.